tools/ncf.py

```diff
@@ -112,16 +112,17 @@ def get_all_techniques_filenames(migrate_technique = False):
   if migrate_technique:
     path = os.path.join(basePath,"ncf/50_techniques")
   else:
-    path = os.path.join(basePath,"techniques/ncf_techniques")
+    path = os.path.join(basePath,"techniques")
 
   return get_all_cf_filenames_under_dir(path, not migrate_technique)
 
-
+excluded_dirs = [ "applications", "fileConfiguration", "fileDistribution", "jobScheduling", "systemSettings" ]
```

Comment:
why not `system` too?

Reply:
because i forgot system, so i guess it's a really good catch, but i prefer to say that I did it on purpose to test you if you really read my code ...

Thank you :)